packages/react-meteor-data/suspense/useTracker.ts

```diff
@@ -0,0 +1,305 @@
+import { Tracker } from 'meteor/tracker'
+import { EJSON } from 'meteor/ejson'
+import { DependencyList, useEffect, useMemo, useReducer, useRef } from 'react'
+import { Meteor } from 'meteor/meteor'
+
+function checkCursor(data: any): void {
+  let shouldWarn = false
+  if (Package.mongo && Package.mongo.Mongo && data && typeof data === 'object') {
+    if (data instanceof Package.mongo.Mongo.Cursor) {
+      shouldWarn = true
+    } else if (Object.getPrototypeOf(data) === Object.prototype) {
+      Object.keys(data).forEach((key) => {
+        if (data[key] instanceof Package.mongo.Mongo.Cursor) {
+          shouldWarn = true
+        }
+      })
+    }
+  }
+  if (shouldWarn) {
+    console.warn(
+      'Warning: your reactive function is returning a Mongo cursor. ' +
+      'This value will not be reactive. You probably want to call ' +
+      '`.fetch()` on the cursor before returning it.'
+    )
+  }
+}
+
+export const cache = new Map<string, Cache>()
+
+interface Cache {
```

Comment:
It's a _cache entry_, not _cache_. Plus it'd be nice to have a similar naming across all suspense hooks.